src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -332,10 +332,9 @@
   },
   "stsjorbsmod:FocusedRage": {
     "NAME": "Focused Rage",
-    "DESCRIPTION": "Gain !stsjorbsmod:UrMagic! Strength for each card stsjorbsmod:Exerted this combat. NL stsjorbsmod:Exert.",
+    "DESCRIPTION": "Gain !stsjorbsmod:UrMagic! Strength for each card stsjorbsmod:Exerted this combat NL (including this card). NL stsjorbsmod:Exert.",
```

Comment:
As discussed, lets change this to: 
`stsjorbsmod:Exert. NL Gain !stsjorbsmod:UrMagic! Strength for each card stsjorbsmod:Exerted this combat.`